mopidy/ext.py

```diff
@@ -1,61 +1,71 @@
-import collections
+from __future__ import annotations
+
 import logging
 from collections.abc import Mapping
+from typing import TYPE_CHECKING, NamedTuple
 
 import pkg_resources
 
 from mopidy import config as config_lib
 from mopidy import exceptions
 from mopidy.internal import path
 
-logger = logging.getLogger(__name__)
+if TYPE_CHECKING:
+    from pathlib import Path
+    from typing import Any, Dict, Iterator, List, Optional, Type, Union
+
+    from mopidy.commands import Command
+    from mopidy.config import ConfigSchema
 
+    ConfigValue = Union[None, bool, float, int, str, int]
```

Comment:
`int` listed twice? Do we need `List[str]`?

Reply:
Attempting to type config values could make it unnecessarily annoying for extensions to utilise configuration. Right now we have the guarantee that the configuration is validated by Mopidy core, and can use the value freely. If this was typed as `Any`, extension code can simply cast the configuration value however they wanted (with the knowledge that that cast is safe) and keep going.

If the range of possible types is locked down, it could make it unnecessarily annoying to assert to the type-checker that you know the actual type. It could also make it even more difficult to use new/custom config types, like the `Pair` type I outlined in #1965.

Reply:
I agree that typing this as `Any` is probably the right choice now, and then leave further strictness to sometime in the future when the config system might be rebuilt around types.